app/konnect/dev-portal/audit-logging/replay-job.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Set up an audit log replay job
+content_type: how-to
+---
+
+You can use the {{site.dev-portal_short_name}} Audit Logs API to configure replay jobs for [audit logging](/konnect/dev-portal/audit-logging/). 
+
+Replay jobs are useful when you have missed audit log entries due to an error or a misconfigured audit
+log webhook. You may have one replay job at a time per region, and request data from up to one week ago.
+A replay job in a region will resend data for the requested timeframe to the webhook configured for that region.
+
+## Prerequisites
+
+* [Org Admin or Portal Admin permissions](/konnect/org-management/teams-and-roles/teams-reference/)
```

Comment:
I think only Org Admins have permissions for audit logs today. We will be adding the permissions for Portal Admins in future iterations. @neethi-shashidhar-kong - can you please confirm?

Reply:
That is correct, only Org Admins have the permissions for now.

Reply:
Thanks! I'll get those prereqs fixed